src/pages/new-vehicle.vue

```
@@ -25,72 +58,205 @@
+// Update the search function to use OIB search
+const fu = async (query: string) => {
+  await searchUserByOIB(query);
   return new Promise((resolve) => {
     setTimeout(() => {
       resolve('Promise resolved');
     }, 2000);
   });
 };
 
+// Add function to handle vehicle data updates
+const handleVehicleDataUpdate = (data: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)